test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetInstallCommandTest.cs

```diff
@@ -2242,6 +2242,12 @@ public async Task InstallCommand_WithASourceThatReportsVulnerabilities_RaisesVul
             File.Exists(a1Nupkg).Should().BeFalse();
         }
 
+        [SkipMono()]
+        public void InstallCommand_WithAuditSource_AndPackageWithVulnerabilities_RaisesWarnings()
+        {
+            NuGetRestoreCommandTest.Command_WithAuditSource_AndPackageWithVulnerabilities_RaisesWarnings("restore");
```

Comment:
is this supposed to be `install`, rather than `restore`?  Otherwise it looks like an exact duplicate of the test added in NuGetRestoreCommandTest.cs